components/pages/Messages/Messages.js

```diff
@@ -174,25 +174,212 @@ export function Messages({ fromUserId }) {
+                                  <circle cx="12" cy="12" r="1.5"></circle>
+                                  <circle cx="6.5" cy="12" r="1.5"></circle>
+                                  <circle cx="17.5" cy="12" r="1.5"></circle>
+                                </svg>
+                              </button>
+                            </div>
+                            <p
+                              className={cx(styles.messageText, {
+                                [styles.isSent]: message.direction === 'sent',
+                              })}
+                              onDoubleClick={() =>
+                                toggleMessageLike(fromUserId, index)
+                              }
+                            >
+                              {message.message_body}
+                            </p>
+                          </div>
+                        ) : (
+                          <div className={styles.messageTextAndThreeDots}>
+                            <p
+                              className={cx(styles.messageText, {
+                                [styles.isSent]: message.direction === 'sent',
+                              })}
+                              onDoubleClick={() =>
+                                toggleMessageLike(fromUserId, index)
+                              }
+                            >
+                              {message.message_body}
+                            </p>
+                            <div className={styles.messageInteraction}>
+                              {messageIndex === index && (
+                                <div
+                                  className={styles.interactButtonsContainer}
+                                >
+                                  <button
+                                    className={styles.interactButtons}
+                                    onClick={() => {
+                                      messageUnLike(fromUserId, index);
+                                      setMessageIndex(undefined);
+                                    }}
+                                  >
+                                    Unlike
+                                  </button>
+                                  <button
+                                    className={styles.interactButtons}
+                                    onClick={() => {
+                                      navigator.clipboard.writeText(
+                                        message.message_body
+                                      );
+                                      setMessageIndex(undefined);
+                                    }}
+                                  >
+                                    Copy
+                                  </button>
+                                  <button className={styles.interactButtons}>
+                                    Report
+                                  </button>
```

Comment:
These buttons don't work, and it's pretty tricky to guess why. Can you try to debug this?

Reply:
Yea, the onBlur is doing it's job before the onClick on the buttons.
Fixed it.